### DISCUSSION TITLE
Drizzle support

### DISCUSSION THREAD

**Discussion by FrimJo** [sqlitecloud/discussions#20] [Ideas]
Hi all 👋🏼

I would love to use SQLite Cloud with [Drizzle ORM](https://orm.drizzle.team/) but I do not know where to start. Seems like Drizzle ORM is using [libsql-client-ts](https://github.com/tursodatabase/libsql-client-ts#supported-urls) under the hood when dialect `sqlite` is set in the config, but I'm getting an error telling me that `sqlitecloud:` is not supported, only `libsql:`, `wss:`, `ws:`, `https:`, `http:` and `file:` are.

I would be glad to receive some pointers on where to start, thnx!

**Comment by btholt** [comment]
Appreciate the ask here. We have some new employees starting soon whose jobs will be to investigate all these integrations. Stay tuned!
